inc/core/styles/frontend.php

```diff
@@ -722,8 +722,8 @@ public function setup_layout_subscribers() {
 
 		$this->_subscribers['.neve-main > .single-post-container .nv-sidebar-wrap'] = [
 			Config::CSS_PROP_MAX_WIDTH => [
-				Dynamic_Selector::META_KEY         => Config::MODS_SINGLE_CONTENT_WIDTH,
-				Dynamic_Selector::META_DEFAULT     => $this->sidebar_layout_width_default( Config::MODS_SINGLE_CONTENT_WIDTH ),
+				Dynamic_Selector::META_KEY         => $this->get_sidebar_content_width_meta( $context, $allowed_context ),
+				Dynamic_Selector::META_DEFAULT     => $this->sidebar_layout_width_default( $this->get_sidebar_content_width_meta( $context, $allowed_context ) ),
 				Dynamic_Selector::META_DEVICE_ONLY => Dynamic_Selector::DESKTOP,
```

Comment:
Can we save these two calls in separate variables at the top? 

It seems they are called multiple times inside this function. I might be missing something, but I don't think the result changes between calls 👀